packages/xo-web/src/xo-app/dashboard/health/index.js

```diff
@@ -533,6 +533,10 @@ const HANDLED_VDI_TYPES = new Set(['system', 'user', 'ephemeral'])
           return false
         }
 
+        if (vdi.name_label === 'PVS cache VDI' && vdi.name_description === 'PVS cache VDI') {
+          return false
+        }
```

Comment:
You can mutualize that with the `if` above.
Also add a comment that link to github issue.
```js
if(
   ... ||
   vdi.name_label === 'PVS cache VDI' && vdi.name_description === 'PVS cache VDI' // see https://github.com/vatesfr/xen-orchestra/issues/7938
 ) {...}
 ```